charts/nexus3/templates/deployment.yaml

```diff
@@ -74,15 +74,13 @@ spec:
             {{- with .Values.env }}
             {{- . | toYaml | trim | nindent 12 }}
             {{- end }}
-          command:
-            - "sh"
-            - "-c"
-          args:
-            - echo "Starting Nexus...";
-              {{- if .Values.config.enabled }}
-              bash /opt/sonatype/nexus/conf/configure.sh;
-              {{- end }}
-              /opt/sonatype/start-nexus-repository-manager.sh;
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - "sh"
+                  - "-c"
+                  - "${SONATYPE_DIR}/nexus/conf/configure.sh"
```

Comment:
I think this should still be wrapped in the `{{- if .Values.config.enabled }}` staement.

Reply:
Yes, that's correct, and I can fix that.

Reply:
Updated.